sqlmesh/schedulers/airflow/dag_generator.py

```diff
@@ -83,6 +85,8 @@ def __init__(
         self._external_table_sensor_factory = external_table_sensor_factory
         self._state_reader = state_reader
         self._sensor_mode = sensor_mode
+        self._high_water_mark_sensor_args = high_water_mark_sensor_args or {}
```

Comment:
Should we remove the `sensor_mode` argument since it can now just be passed as part of arguments?

Reply:
One reason why why `sensor_mode` would be treated different than the rest is because it is changing the default behavior (default is `poke`). That is why I initially kept it at least since it is making explicit/obvious this behavior change that SQLMesh is doing.